synapse/tests/test_lib_stormlib_modelext.py

```diff
@@ -49,6 +52,38 @@ async def test_lib_stormlib_modelext(self):
             self.none(core.model.prop('_visi:int:tick'))
             self.none(core.model.tagprop('score'))
 
+            # Verify extended forms can't be deleted if they have associated extended props
+            await core.callStorm(create)
+
+            self.nn(core.model.form('_visi:int'))
+            self.nn(core.model.prop('_visi:int:tick'))
+
+            q = '$lib.model.ext.delForm(_visi:int)'
+            with self.raises(s_exc.CantDelForm) as exc:
+                await core.callStorm(q)
+                self.eq('Form has extended properties: tick', exc.exception.get('mesg'))
```

Comment:
The exception would be raises here before the `self.eq()` statement is executed. If you remove one level of indentation you can access the `.exception` property on the context manager to make test assertions on its contents.